scripts/benchmarking.sh

```diff
@@ -43,12 +43,12 @@ function choose_and_bench {
 
 function bench {
     echo "benchmarking '${1}::${2}' --check=${3}"
-    if [[ "${check}" -eq 1 ]]; then
+    if [[ "${3}" -eq 1 ]]; then
```

Comment:
Is this part necessary?

Reply:
This was a bug in script, there's no `check` variable in scope. The line above demonstrates it.